### PR TITLE
Feature/docker compose

### PR DESCRIPTION
**Pull request by chinook25**

### CODE REVIEW

File: server/docker/Dockerfile [hunk 1]
```diff
@@ -36,4 +36,4 @@ RUN unzip patavi.zip && mv patavi-*/server/* .
 
 RUN cd public && yarn
 
-CMD ["pm2", "start", "--no-daemon", "server.js"]
+CMD ["node", "server.js"]
```
Comment (chinook25) [inline server/docker/Dockerfile]: I think this should not be changed, as this is the 'normal' dockerfile for patavi

Comment (kzalite) [reply server/docker/Dockerfile]: Agree

File: .github/workflows/master-push-build-image-server.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+name: Patavi server | push | build docker compose container
```
Comment (DanielReid) [inline .github/workflows/master-push-build-image-server.yml:1]: name should include `| push tag |` for clarity I think

File: .github/workflows/master-push-build-image-server.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          echo "${{ secrets.pataviComposeCA }}" > server/ssl/ca-crt.pem
+
+      - name: build docker image
+        run: |
+          cd server
+          cp dockerCompose/Dockerfile .
+          docker build --tag registry.drugis.org/patavi-server:composed .
+
+      - name: docker login
+        run: docker login -u drugis -p ${{ secrets.dockerDrugisRegistryPassword }} registry.drugis.org
```
Comment (DanielReid) [inline .github/workflows/master-push-build-image-server.yml:36]: shouldn't this go to the public docker registry?

File: .github/workflows/master-push-update-worker-base-image.yml [hunk 4]
```diff
@@ -2,7 +2,8 @@ name: Patavi | push | worker build & push
 
 on:
   push:
-    branches: master
+    tags:
```
Comment (DanielReid) [inline .github/workflows/master-push-update-worker-base-image.yml]: think this contradicts the filename?

**Pull request merged**